spec/datadog/tracing/validate_benchmarks_spec.rb

```diff
@@ -16,7 +16,7 @@
   benchmarks_to_validate.each do |benchmark|
     describe benchmark do
       it 'runs without raising errors' do
-        expect_in_fork do
+        expect_in_fork(timeout_seconds: 30) do
```

Comment:
I encountered a timeout failure in https://github.com/DataDog/dd-trace-rb/pull/4207 but I thought my benchmark was taking too long (which still may be the case).

My understanding is the current timeout is 10 seconds, perhaps raising it to 20 would be sufficient rather than going to 30? What do you think?